Src/DependencyCollector/NetCore.Tests/DependencyTrackingTelemetryModuleTestNetCore.cs

```diff
@@ -110,60 +110,114 @@ public void Cleanup()
         [Timeout(5000)]
         public async Task TestDependencyCollectionWithParentActivity()
         {
-            ITelemetry sentTelemetry = null;
-
-            var channel = new StubTelemetryChannel
-            {
-                OnSend = telemetry =>
-                {
-                    // The correlation id lookup service also makes http call, just make sure we skip that
-                    DependencyTelemetry depTelemetry = telemetry as DependencyTelemetry;
-                    if (depTelemetry != null &&
-                        !depTelemetry.Data.StartsWith(FakeProfileApiEndpoint, StringComparison.OrdinalIgnoreCase))
-                    {
-                        Assert.IsNull(sentTelemetry);
-                        sentTelemetry = telemetry;
-                    }
-                },
-                EndpointAddress = FakeProfileApiEndpoint
-            };
-
-            var config = new TelemetryConfiguration
-            {
-                InstrumentationKey = IKey,
-                TelemetryChannel = channel
-            };
-
-            config.TelemetryInitializers.Add(new OperationCorrelationTelemetryInitializer());
-
             using (var module = new DependencyTrackingTelemetryModule())
             {
-                module.ProfileQueryEndpoint = FakeProfileApiEndpoint;
-                module.Initialize(config);
+                module.Initialize(this.config);
 
                 var parent = new Activity("parent").AddBaggage("k", "v").SetParentId("|guid.").Start();
 
-                var url = new Uri("http://bing.com");
+                var url = new Uri(localhostUrl);
                 var request = new HttpRequestMessage(HttpMethod.Get, url);
-                await new HttpClient().SendAsync(request);
+                using (new LocalServer(localhostUrl))
+                {
+                    await new HttpClient().SendAsync(request);
+                }
 
-                // on netcoreapp1.0 DiagnosticSource event is fired asycronously, let's wait for it 
+                // on netcoreapp1.0 DiagnosticSource event is fired asychronously, let's wait for it 
```

Comment:
"Asynchronously" is still not quite right.
Same nitpick as above about sentence format.